Static/js/chat.js

```diff
@@ -1,265 +1,271 @@
+		}
+		else { // Assuming priv
+			server.chat('priv', this.cur_room.split('priv-')[1], msg);
+		}
+
+		$('#chat-me').val('');
+	},
+	/*
+		 * Callback from UI
+		 */
+	selectRoom: function (type, name) {
+		this.cur_room = (`${type}-${name}`);
+		if (type === 'global') {
+			this.cur_room = 'global';
+		}
+
+		const title = $(`.room-name-${this.cur_room} a span`).html();
+		$('#cur_room').html(title);
+	},
+
+	setRoom: function (type, name) {
+		const room = `${type}-${name}`;
+		$(`.room-name-${room} a`).tab('show');
+		chathandler.selectRoom(type, name);
+	},
+	/*
+		 * Callback from UI
+		 */
+	removeRoom: function (type, name) {
+		const room = `${type}-${name}`;
+		console.log(`remove ${room}`);
+
+		if (this.cur_room === room) {
+			$('#room_list li:eq(0) a').tab('show');
+			this.selectRoom('global', 'global');
+		}
+
+		if (type === 'room') {
+			server.leaveroom(name);
+		}
+
+		$(`.room-name-${room}`).remove();
+		$(`#room-div-${room}`).remove();
+	},
+	/*
+		 * Callback from UI
+		 */
+	createRoom: function (type, name, title) {
+		const room = `${type}-${name}`;
+
+		const roomDiv = $('<div/>').attr('id', `room-div-${room}`)
+			.addClass('tab-pane');
+		$('#room_divs').append(roomDiv);
+
+		const roomList = $('#room_list');
+		const a = $('<a/>').click(() => chathandler.selectRoom(type, name))
+			.attr('data-toggle', 'tab')
+			.attr('href', `#room-div-${room}`)
+			.append(title);
+		const li = $('<li/>').append(a).addClass(`room-name-${room}`);
+
+		$('<div/>').addClass('btn').html('&times;')
+			.click(() => chathandler.removeRoom(type, name))
+			.appendTo(li);
+
+		$('#room_list').append(li);
+	},
+
+	/*
+		 * Callback from UI
+		 */
+	createGameRoom: function (game, p1, p2) {
+		const p1span = $('<span/>').html(p1).addClass('playername');
+		const p2span = $('<span/>').html(p2).addClass('playername');
+		const vs = $('<span/>').html(' vs ');
+		const sp = $('<span/>').append(p1span).append(vs).append(p2span);
+
+		this.createRoom('room', game, sp);
+	},
+
+	createPrivateRoom: function (player) {
+		const psp = $('<span/>').html(player).addClass('playername');
+		const sp = $('<span/>').append(psp);
+
+		this.createRoom('priv', player, sp);
+	},
+
+	roomExists: function (type, name) {
+		return $(`.room-name-${type}-${name}`).length;
+	},
+	/*
+		 * Notify checkbox change for checkbox:
+		 * Hide chat time
+		 */
+	hideChatTime: function () {
+		if (document.getElementById('hide-chat-time').checked) {
+			localStorage.setItem('hide-chat-time', 'true');
+			$('.chattime').each(function (index) {
+				$(this).addClass('hidden');
+			});
+		} else {
+			localStorage.setItem('hide-chat-time', 'false');
+			$('.chattime').each(function (index) {
+				$(this).removeClass('hidden');
+			});
+		}
+	},
+	showchat: function () {
+		$('#chat-toggle-button').css('right', this.chat_width + 5);
+		$('#chat-toggle-text').html('>><br>c<br>h<br>a<br>t');
+		$('#chat').removeClass('hidden');
+		if (fixedcamera || true) {
```

Comment:
@NoHatCoder Do you know why this check exists here and in line 216? `|| true` looks like someone wanted to temporarily disable the condition - but if that's a permanent thing, we could remove it.

Reply:
I wrote this while I was still unsure how all the cameras would work, I think we can just remove the if now.